pkg/activator/handler/handler.go

```diff
@@ -61,7 +72,45 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	err := a.Throttler.Try(revID, func() {
-		attempts, httpStatus := a.proxyRequest(w, r, target)
+		var attempts int
+		var httpStatus int
+		// If a GET probe interval has been configured, then probe
+		// the queue-proxy with our network probe header until it
+		// returns a 200 status code.
+		success := (a.GetProbeCount == 0)
+		probeReq := &http.Request{
```

Comment:
Do we make sure that the request is not delivered to the user container even if we remove istio sidecar injection?

Sorry, if it was discussed somewhere else. Maybe it is covered somewhere and I missed it.

Reply:
There was a separate change that added handling of this to the queue-proxy and activator.